logbook-spring-boot-autoconfigure/src/main/java/org/zalando/logbook/autoconfigure/LogbookAutoConfiguration.java

```diff
@@ -304,6 +304,12 @@ public HttpLogWriter writer() {
         return new DefaultHttpLogWriter();
     }
 
+    @Bean
+    @ConditionalOnMissingBean(LogbookClientHttpRequestInterceptor.class)
+    public LogbookClientHttpRequestInterceptor logbookClientHttpRequestInterceptor(Logbook logbook) {
```

Comment:
How would a usual integration look like on the user side? Inject this by hand and call a `RestTemplateBuilder`?

Reply:
With the builder, should look like

```java
private final RestTemplate rt;
MyClient(RestTemplateBuilder builder, LogbookClientHttpRequestInterceptor interceptor){
  this.rt = builder
    .additionalInterceptors(interceptor)
    .build();
}
```

Reply:
Should we add that to the README as an example?

Reply:
that makes sense but it'll go under a `spring-boot` section since that's where `RestTemplateBuilder` comes from. I didn't make that assumption for ppl pulling the spring dependency directly